recipes/configs/llama3/70B_generation_distributed.yaml

```diff
@@ -0,0 +1,50 @@
+# Config for running the InferenceRecipe in dev/generate_v2.py to generate output
+# using a Llama3 70B Instruct model
+#
+# This config assumes that you've run the following command before launching:
+#  tune download meta-llama/Meta-Llama-3-70B-Instruct --output-dir /tmp/Meta-Llama-3-70B-Instruct --ignore-patterns "original/consolidated*" --hf-token <HF_TOKEN>
+#
+# To launch, run the following command from root torchtune directory:
+#    tune run --nproc_per_node 8 dev/generate_v2_distributed --config llama3/70B_generation_distributed
+
+output_dir: ./
+
+# Model arguments
+model:
+  _component_: torchtune.models.llama3.llama3_70b
+
+tensor_parallel_plan:
```

Comment:
I think we might want to mirror the parrelilize_module API and call this `parallelize_plan`. See https://pytorch.org/docs/main/distributed.tensor.parallel.html#torch.distributed.tensor.parallel.parallelize_module